client/src/pages/Home.css

```
@@ -9,3 +9,46 @@
 .message:hover {
 	background-color: yellow;
 }
+
+.flex-container {
+  display: flex;
+  flex-direction: column;
+  background-color: rgb(214, 227, 243);
+  align-items: center;
+  height: 100%;
+  border-radius: 12px;
+  padding: 100px 100px 100px 100px;
```

We usually prefer em/vh/vw over pixels, but it can stay for now.

---

```suggestion
  padding: 6rem 1rem;
```
Can we use relative units here? Also, the horizontal padding does not need to be as large so we could use something like `1rem`